Jonathan-jonathankern/Homework-4/99-bottles-of-beer/js/app.js

```diff
@@ -1,22 +1,21 @@
 'use strict';
 
-// create li element
-const createListItemElement = document.createElement('li');
-
 // find ul
 const findUnorderedListElement = document.querySelector('ul');
```

Comment:
As with the `createListItemElement` name below, I'd advocate for dropping the verb here. If you want to preserve an indication of your program flow, adding a comment before the line describing what you're doing would let you still note what you're doing. But I love that you're working toward being descriptive in your code! Keep it up!